web/src/components/merchant/sign-up/SignUpCard.tsx

```diff
@@ -14,7 +14,15 @@
  * limitations under the License.
  */
 
-import React from 'react';
+import {
+  NAME_EMPTY,
+  EMAIL_EMPTY,
+  PASSWORD_EMPTY,
+  PASSWORDS_DO_NOT_MATCH,
+  VPA_EMPTY,
```

Comment:
Are these all the entries we need to onboard a merchant? How about phone number?
What is the identifier for this merchant? Which field is allowed to be null and why?

Reply:
Yes, those are all the entries needed. Actually, I'm not sure if we still need to collect the user's VPA since we are no longer using Spot Payments API.
I don't see why we would need a phone number. But did I miss something?
The identifier would be the email.
None of the fields are allowed to be null because everything is needed.